runner/tests/test_trigger.py

```diff
@@ -115,9 +115,29 @@ def test_operator_trigger_executes_runs_individually(self, create_jobs_from_chai
                  parent=str(operator_run.id)
                  )
         ]
-
         create_jobs_from_chaining.delay.assert_has_calls(calls, any_order=True)
         process_triggers()
         operator_run.refresh_from_db()
         self.assertEqual(operator_run.status, RunStatus.COMPLETED)
 
+    @patch('runner.tasks.create_jobs_from_operator')
+    def test_operator_trigger_passes_request_id_tag_if_one_is_not_provided(self, create_jobs_from_operator):
```

Comment:
should this be called "_if_one_is_provided" ?

Reply:
i meant as in not provided to OperatorFactory, i can clarify